Cargo.lock

```diff
@@ -52,15 +52,15 @@ dependencies = [
 
 [[package]]
 name = "autocfg"
-version = "1.1.0"
+version = "1.3.0"
```

Comment:
This seems to have a bunch of unrelated `Cargo.lock` changes, can we minimize this to what is necessary for tempfile?

Reply:
Okay, changes are now limited to the necessary changes only. If I remember right, I updated everything after the Winows or Mac pipeline failed to compile. Let's hope it works now. Those OSes are not exactly readability available and cross-compiling is still one of Rust's weaknesses.

Reply:
Seems like CI is happy!